adapters/colossus/colossus.go

```diff
@@ -122,18 +118,22 @@ func (a *ColossusAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externa
 }
 
 func getMediaTypeForImp(impID string, imps []openrtb2.Imp) (openrtb_ext.BidType, error) {
```

Comment:
There is one thing for you to consider. I case if imp has multi format this function may return incorrect bid type. We recommend to extract mediaType from response. Here is the example for reference: https://github.com/prebid/prebid-server/blob/74d2199c7dd7e6b2c28d67c26cda2c4405ca5b6f/adapters/algorix/algorix.go#L190

Reply:
thanks, i have updated